src/OrchardCore.Modules/OrchardCore.Tenants/Drivers/FeatureProfileDisplayDriver.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using Newtonsoft.Json;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Environment.Shell.Models;
+using OrchardCore.Mvc.ModelBinding;
+using OrchardCore.Tenants.Models;
+using OrchardCore.Tenants.Services;
+using OrchardCore.Tenants.ViewModels;
+
+namespace OrchardCore.Tenants.Drivers
+{
+    public class FeatureProfileDisplayDriver : DisplayDriver<FeatureProfile>
```

Comment:
Shapes instead of Views ... What is the intent? Scenarios where features can be enhanced by other modules?

Reply:
What if a billing module want to attach a price to a profile? Like if one has a subscription based app, one can charge the tenant based on the available profiles.

Reply:
removed the shapes and used views